src/integration/java/io/pinecone/helpers/TestIndexResourcesManager.java

```diff
@@ -37,13 +39,17 @@ public class TestIndexResourcesManager {
     private Pinecone pineconeClient;
     private String podIndexName;
     private IndexModel podIndexModel;
-    private String serverlessIndexName;
+    private IndexModel serverlessIndexModel;
+    private  String serverlessIndexName;
     private String collectionName;
     private CollectionModel collectionModel;
-    private final List<String> podIndexVectorIds = Arrays.asList("id1", "id2", "id3");
+    private final List<String> vectorIds = Arrays.asList("id1", "id2", "cidddd3");
+    private final String namespace = "example-namespace";
```

Comment:
For most of our data plane tests we create a namespace in the test file and use that for the specific test. I think it should be fine to define a namespace at this level.

We should create a getter for this though, so any consuming tests have access to the namespace that data was seeded under, so I'd look at adding that.

Reply:
It hasn't seemed to _break_ anything, so I think it's okay! But yeah great idea re: the getter method. Adding that now.